src/discussions/comments/reply/Reply.jsx

```diff
@@ -4,13 +4,51 @@ import { useDispatch, useSelector } from 'react-redux';
 import * as timeago from 'timeago.js';
 
 import { injectIntl, intlShape } from '@edx/frontend-platform/i18n';
+import { Avatar, Icon, IconButton } from '@edx/paragon';
+import { MoreVert } from '@edx/paragon/icons';
 
 import { commentShape } from '../comment/Comment';
 import CommentIcons from '../comment-icons/CommentIcons';
 import { selectCommentResponses } from '../data/selectors';
-import { fetchCommentResponses } from '../data/thunks';
+import { editComment, fetchCommentResponses } from '../data/thunks';
 import messages from '../messages';
 
+function ReplyHeader({ reply, intl }) {
+  return (
+    <div className="d-flex flex-row justify-content-between">
+      <div className="align-items-center d-flex flex-row">
+        <Avatar className="m-2" alt={reply.author} src={reply.users[reply.author].profile.image.image_url_small} />
+        <div className="status small">
+          <a href="#nowhere">
```

Comment:
what is the purpose of this

Reply:
The linter fails it otherwise but I couldn't find any obvious way of getting the user's profile.

Reply:
will check it and get back to you.